Makefile

```diff
@@ -249,10 +249,13 @@ install: build
 
 .PHONY: image
 image:
+	cp ./build.assets/charts/Dockerfile $(BUILDDIR)/
+	cd $(BUILDDIR) && docker build . -t quay.io/gravitational/teleport:$(VERSION)
```

Comment:
It is common practice to place path('.' in this case) at the end of the command. At least in our repositories, because it is more readable.